x-pack/test/functional/apps/dashboard_mode/dashboard_empty_screen.js

```diff
@@ -40,18 +40,21 @@ export default function ({ getPageObjects, getService }) {
         operation: 'date_histogram',
         field: '@timestamp',
       });
+      await PageObjects.lens.closeDimensionEditor();
```

Comment:
I suspect you had to put these because the flyout does not close on outside click?

Reply:
I actually removed the line that was closing the dimension from the `configureDimension` method:
https://github.com/elastic/kibana/pull/77863/files#diff-593978ac44529e71d36a3c038cb91c00R113

I was having doubts here – I've done it because in two cases [here](https://github.com/elastic/kibana/pull/77863/files#diff-7b486deaa668360642958fba514193cfR160) and [here](https://github.com/elastic/kibana/pull/77863/files#diff-7b486deaa668360642958fba514193cfR80) I want to keep modifying the dimension after configuring it so I've decided to not close it inside the method.

Reply:
Maybe the `configureDimention` could have a `params` object as argument to disable the closing of the panel? This will reduce a bit the code here and you would be still able to do things manually

Reply:
I'll merge it for now so the branch is not stale during my time off. Will address this in the next PR.